### PR TITLE
i18n - extracting new, pulling from tx

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: locale/en/hammer-cli-foreman-webhooks.po [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=; plural=;\n"
-"\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
-msgid "File containing X509 Certification Authorities concatenated in PEM format"
+msgid ""
+"File containing X509 Certification Authorities concatenated in PEM format"
```
Comment (adamruzicka) [inline locale/en/hammer-cli-foreman-webhooks.po:21]: Uhmm, this looks suspicious

File: locale/en/hammer-cli-foreman-webhooks.po [hunk 2]
```diff
@@ -1,23 +1,24 @@
-# English translations for hammer-cli-foreman-webhooks package.
-# Copyright (C) 2023 THE PACKAGE'S COPYRIGHT HOLDER
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the hammer-cli-foreman-webhooks package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (adamruzicka) [inline locale/en/hammer-cli-foreman-webhooks.po:4]: What went wrong here?

File: locale/en/hammer-cli-foreman-webhooks.po (hunk 1 above)
Comment (ofedoren) [reply locale/en/hammer-cli-foreman-webhooks.po:21]: I've noticed that too, but I've seen that few times, so I mostly ignore that. Maybe I shouldn't...

On the other hand, we pulled fully translated file for Georgian language, where it looks similar.

Comment (adamruzicka) [reply locale/en/hammer-cli-foreman-webhooks.po:21]: This seems to be a multiline syntax so all should be good here

File: locale/en/hammer-cli-foreman-webhooks.po (hunk 2 above)
Comment (ofedoren) [reply locale/en/hammer-cli-foreman-webhooks.po:4]: I have no idea, maybe some meta info got broken on Transifex itself?

Comment (adamruzicka) [reply locale/en/hammer-cli-foreman-webhooks.po:4]: It seems to be similarly broken in other repos we have so let's just roll with it.

**Review by adamruzicka — APPROVED** [review]

**Comment by adamruzicka** [comment]
Thank you @ofedoren !

**Pull request merged**